src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -117,4 +114,13 @@ def put(self, request: Request, project, filter_id) -> Response:
             data={"state": returned_state},
         )
 
-        return Response(status=204)
+        data = {
+            "id": current_filter.id,
+            # 'active' will be a list for the legacy browser filter.
+            # All other filters will be boolean
+            "active": list(new_state) if isinstance(new_state, Iterable) else new_state,
+            "description": flt.description,
+            "name": flt.name,
```

Comment:
Ideally this would match the response for the filters GET `ProjectFiltersEndpoint` which has the extra `hello` field formatted like:
`"hello": flt.id + " - " + flt.name,`